ros_buildfarm/templates/release/rpm/mock_config.cfg.em

```diff
@@ -0,0 +1,50 @@
+include('/etc/mock/default.cfg')
+
+# Change the root name since we're modifying the chroot
+config_opts['root'] += '-ros-buildfarm'
+
+# Add python3-rpm-macros to resolve %{python3_pkgversion} in rosdep rules
+config_opts['chroot_setup_cmd'] += ' python3-rpm-macros'
```

Comment:
I would rather store the command as a list to not be limited to `shell=True` execution.

Reply:
I'm appending to the existing command, which is defined here: https://github.com/rpm-software-management/mock/blob/8d0c3ad5ddc36d4f7649cc1097de003db2219a0b/mock-core-configs/etc/mock/templates/fedora-31.tpl#L5

I can't append a list to a string, and converting the existing value to a list just to match our pattern seems like a bad idea, given that there could be other locations in `mock` where the command is assumed to be a string.